src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -582,6 +582,11 @@ uct_rc_mlx5_ep_check(uct_ep_h tl_ep, unsigned flags, uct_completion_t *comp)
     UCT_CHECK_PARAM(comp == NULL, "Unsupported completion on ep_check");
     UCT_CHECK_PARAM(flags == 0, "Unsupported flags: %x", flags);
 
+    if (ucs_unlikely(!(ep->super.flags & UCT_RC_EP_FLAG_CONNECTED))) {
```

Comment:
add ucs_assert(ep->super.flags & UCT_RC_EP_FLAG_CONNECTED) to other send funcs

Reply:
- convert this flag to assertion only
in next phase:
- change wireup_ep forward ep_check to aux lane ep_check (what if it doesn't support it)?
- add test with small KA interval to reproduce an issue of sending KA on unconnected endpoint

Reply:
added assert into CHECK_RES macro